google-cloud-spanner/lib/google/cloud/spanner/results.rb

```diff
@@ -85,6 +86,8 @@ def rows
 
           fields = @metadata.row_type.fields
           values = []
+          buffered_responses = []
+          buffer_upper_bound = 10
```

Comment:
This is simply the method used by node. I also haven't seen any direction in the requirements document about how to configure the buffer, or how to determine if its full. Is it the number of requests? Is it the total size of data in the buffer? I don't know.